packages/manager/modules/hub/src/components/products/translations/Messages_fr_FR.json

```
@@ -28,5 +28,7 @@
   "manager_hub_products_VEEAM_VEEAM_ENTERPRISE": "Veeam Enterpise",
   "manager_hub_products_VPS": "VPS",
   "manager_hub_products_VRACK": "vRacks",
-  "manager_hub_products_XDSL": "xDSL"
+  "manager_hub_products_XDSL": "xDSL",
+  "manager_hub_products_see_less": "Voir moins de service",
```

Isn't supposed to be:
```suggestion
  "manager_hub_products_see_less": "Voir moins de services",
```